content/docs/1.2.3/deploy/install/_index.md

```diff
@@ -86,14 +86,42 @@ If MountPropagation is disabled, Base Image feature will be disabled.
 
 ### Root and Privileged Permission
 
-The longhorn manager uses `Bidirectional` mount propagation to access the host's disk (default `/var/lib/longhorn`) in the container.
-An use case is to access disk metadata and replicas directory on the host.
-
-Besides, Longhorn uses `nsenter` command to execute commands on the host. The `nsenter` command, for example, is to check the host's disk space usage
-```shell
-stat /var/lib/longhorn/ -fc '{"path":"%n","fsid":"%i","type":"%T","freeBlock":%f,"totalBlock":%b,"blockSize":%s}'
-```
-The `nsenter` command rely on privileged containers and `hostPID: true`.
+Longhorn components require root access with privileged permissions to achieve volume operations and management, because Longhorn relies on system resources on the host across different namespaces, for example, Longhorn uses `nsenter` to understand block devices' usage or encrypt/decrypt volumes on the host.
+
+Below are the list of the directories of all components are:
+
+- Longhorn Manager
+  - /dev: Volume block device
+  - /proc: To access host's mount namespace by `nsenter`
+  - /var/lib/longhorn: Data path
+- Longhorn Engine
+  - /var/lib/longhorn/engine-binaries: Longhorn engine binary
+- Longhorn Instance Manager (Engine)
+  - /dev: Volume block device
+  - /proc: To access host's mount namespace
+  - /var/lib/longhorn/engine-binaries: Longhorn engine binary
+- Longhorn Instance Manager (Replica)
+  - /: Data path (allow the user to customize data path)
+- Longhorn Share Manager
+  - /dev: Volume block device
+  - /lib/modules: volume encryption by `cryptsetup`
+  - /proc: To access host's mount namespace by `nsenter`
+  - /sys: Volume encryption by `cryptsetup`
+- Longhorn CSI Plugin
+  - /: For RWX shared volume
+  - /dev: Volume block device
+  - /lib/modules: Kubernetes CSI plugin
+  - /sys: Volume encryption by `cryptsetup`
+  - /var/lib/kubelet/plugins/kubernetes.io/csi: Kubernetes CSI plugin
+  - /var/lib/kubelet/plugins_registry: Kubernetes CSI plugin
+  - /var/lib/kubelet/plugins/driver.longhorn.io: Kubernetes CSI plugin
+  - /var/lib/kubelet/pods: Kubernetes CSI plugin
```

Comment:
Not sure if we need to explain this part a little bit more.

Reply:
Probably we should.  The intention is to let user understand why the CSI compatible storage solution like longhorn need these privileged permissions besides our components. However, the general explanation is good enough.